checks.d/go_expvar.py

```diff
@@ -153,7 +148,7 @@ def parse_expvar_data(self, data, tags, metrics, max_metrics, namespace):
 
                 if count >= max_metrics:
                     self.warning("Reporting more metrics than the allowed maximum. "
-                                 "Please contact support@datadoghq.com for more information.")
+                                 "Please contact support@stackstate.com for more information.")
```

Comment:
is it support@stackstate.com? or info@stackstate.com? please just double check our support url